packages/appChrome/components/AppChrome.tsx

```diff
@@ -8,55 +8,52 @@ import {
   flush
 } from "../../shared/styles/styleUtils";
 import { ThemeProvider } from "@emotion/react";
-import getCSSVarValue from "../../utilities/getCSSVarValue";
-import { themeBgPrimaryInverted } from "../../design-tokens/build/js/designTokens";
 import {
   defaultSidebarItemHorizPaddingSize,
   defaultSidebarItemVertPaddingSize
 } from "./SidebarItem";
+import {
+  defaultBgColor,
+  defaultHeaderPaddingHor,
+  defaultHeaderPaddingVert
+} from "./HeaderBar";
 
 export interface AppChromeProps {
   sidebar: React.ReactNode;
   headerBar: React.ReactNode;
   mainContent: React.ReactNode;
 }
 
-class AppChrome extends React.PureComponent<AppChromeProps, {}> {
-  public render() {
-    const { sidebar, headerBar, mainContent } = this.props;
-
-    return (
-      <ThemeProvider
-        theme={{
-          sidebarBackgroundColor: getCSSVarValue(themeBgPrimaryInverted),
-          sidebarItemPaddingHor: defaultSidebarItemHorizPaddingSize,
-          sidebarItemPaddingVert: defaultSidebarItemVertPaddingSize
-        }}
+const AppChrome = ({ sidebar, headerBar, mainContent }: AppChromeProps) => {
+  return (
+    <ThemeProvider
+      theme={{
+        sidebarItemPaddingHor: defaultSidebarItemHorizPaddingSize,
+        sidebarItemPaddingVert: defaultSidebarItemVertPaddingSize,
+        headerPaddingHor: defaultHeaderPaddingHor,
+        headerPaddingVert: defaultHeaderPaddingVert,
+        headerBackgroundColor: defaultBgColor
+      }}
+    >
+      <div
+        className={cx(appChrome, textSize("m"), flex({ direction: "column" }))}
+        data-cy="appChrome"
       >
-        <div
-          className={cx(
-            appChrome,
-            textSize("m"),
-            flex({ direction: "column" })
-          )}
-          data-cy="appChrome"
-        >
-          <div data-cy="headerBar">{headerBar}</div>
-          <div className={cx(flex(), appWrapper)}>
-            <div className={flexItem("shrink")} data-cy="sidebar">
-              {sidebar}
-            </div>
-            <main
-              className={cx(flexItem("grow"), flush("left"), appWrapper)}
-              data-cy="main"
-            >
-              {mainContent}
-            </main>
+        <div data-cy="headerBar">{headerBar}</div>
+        <div className={cx(flex(), appWrapper)}>
+          <div className={flexItem("shrink")} data-cy="sidebar">
+            {sidebar}
           </div>
+          <main
+            className={cx(flexItem("grow"), flush("left"), appWrapper)}
+            data-cy="main"
+          >
+            {mainContent}
+          </main>
         </div>
-      </ThemeProvider>
-    );
-  }
-}
+      </div>
+    </ThemeProvider>
+  );
+};
 
-export default AppChrome;
+export default React.memo(AppChrome);
```

Comment:
I feel like memoizing the whole app might have some repercussions. Why do we need to do this? It might be worth running the React DevTools Profiler with and without memo and seeing the difference in performance.

Reply:
The reason I've done this is that when I've refactored another component that was a `PureComponent `it was suggested that I should memoize it to retain that functionality.

Reply:
Removing this after further review, since we are not doing any expensive calculations with this component there will not be a big difference in performance.